_includes/cloudcode/cloud-code.md

```diff
@@ -672,6 +672,9 @@ Parse.Cloud.afterFind(Parse.User, async (request) => {
 })
 ```
 
+### Some considerations to be aware of
+- If you use the `masterKey` to fetch a pointer in an `afterFind` trigger, it will be sent in full to the client. Be sure to check that the returned objects and pointers are secured prior to returning to the client.
```

Comment:
"the returned objects and pointers are secured" this is a bit ambiguous, I think 'secured' could make people think they should be changing ACLs etc.

Perhaps something like "Prior to returning to the client, be sure to check that the returned objects and pointers do not contain information that the client should not be able to access."